test/src/unit-uri.cc

```diff
@@ -74,8 +74,12 @@ TEST_CASE("URI: Test file URIs", "[uri]") {
   CHECK(!uri.is_invalid());
   CHECK(URI::is_file(uri.to_string()));
   CHECK(uri.to_string() == "file:///path");
+#ifndef _WIN32
+  // note: "file://path" is an accepted URI form on windows for UNC
+  // level share viewing,but expected .is_invalid() on *nx.
```

Comment:
`*nx` -> `*nix`

Reply:
Done (and a space added after a comma in same comment.)